src/main/kotlin/no/skatteetaten/aurora/boober/platform/WebPlattformHandler.kt

```diff
@@ -0,0 +1,60 @@
+package no.skatteetaten.aurora.boober.platform
+
+import com.fkorotkov.kubernetes.containerPort
+import com.fkorotkov.kubernetes.envVar
+import no.skatteetaten.aurora.boober.mapper.AuroraConfigFieldHandler
+import no.skatteetaten.aurora.boober.model.Container
+import no.skatteetaten.aurora.boober.utils.addIfNotNull
+
+class WebPlattformHandler : AbstractHandler() {
+    override val container: List<Container>
+        get() = listOf(
+                Container("node", listOf(
+                        containerPort {
+                            containerPort = 9090
```

Comment:
To avoid duplicated declarations of these ports they should be declared as constants in the class header and referenced  instead.

Reply:
That is a possibility. However do we always want there to be an ENV var that corresponds to the port? Should we declare the ports are containerPort (kubernetes) type object here or some intermediary format?